UnitTests/OIDTokenRequestTests.m

```diff
@@ -186,6 +186,12 @@ - (void)testCopying {
                         kTestAdditionalParameterValue,
                         @"The request's kTestAdditionalParameterKey additional parameter should "
                         "be equal to kTestAdditionalParameterValue.");
+  XCTAssertNotNil(request.additionalHeaders,
+                        @"Request's additionalHeaders field should not be nil.");
```

Comment:
Please fix alignment.

Reply:
Not sure if I see what alignment should be fixed here.

![image](https://github.com/openid/AppAuth-iOS/assets/506722/e9187eaa-ef12-427c-a4b6-105ceed9911e)

Reply:
I've got this wrong then. I thought it should've been the other way around. I updated the two wrong formats in the latest commit.